ci/docker-rust-nightly/Dockerfile

```diff
@@ -10,4 +10,8 @@ RUN set -x \
  && cargo --version \
  && cargo install grcov \
  && rustc +nightly-$date --version \
- && cargo +nightly-$date --version
+ && cargo +nightly-$date --version \
+ # codecov
+ && curl -Os https://uploader.codecov.io/latest/linux/codecov \
+ && chmod +x codecov \
+ && mv codecov /usr/bin
```

Comment:
i think we can forgo verifying the published sig of the binary, alrought this practice is rightfully advised according codecov doc.

maybe, we can do that bonus rigidity in distance future.. ;) at least, our security isn't degraded from good-old `curl | bash`. ;)